README.md

```diff
@@ -92,43 +173,55 @@ additional options, for instance, to see the logs from the supervisor service:
 ./dindctl logs -fn 100 -u resin-supervisor
 ```
 
-### Stop the supervisor
+#### Stop the supervisor
 
 ```bash
 ./dindctl stop
 ```
 
-This will stop the container and remove it, also removing its volumes.
-
-## Developing with a balenaOS device
+This will stop the container and remove it, also removing
+its volumes.
 
-If you want to test local changes (only changes to the Node.js code are supported) on a real balenaOS device, provision
-a [development OS image](https://docs.balena.io/understanding/understanding-devices/2.0.0/#dev-vs-prod-images) and power up the device. On the balenaCloud dashboard, take note of the device's IP address. Then run:
+## Developing using a production image or device
 
-```
-./sync.js <device IP>
-```
+A production balena image does not have an open docker
+socket, required for livepush to work. In this situation,
+the `tools/sync.js` script can be used. Note that this
+process is no longer actively developed, so your mileage may
+vary.
 
-This will build the supervisor code and sync it onto the running supervisor container inside the device, and then restart it.
+Bug reports and pull requests are still accepted for changes
+to `sync.js`, but the balenaSupervisor team will focus on
+`npm run sync` in the future.
```

Comment:
I would drop this bit entirely - we should follow this PR up with removing the `sync.js` method

Reply:
There is still a valid time to use it, that we don't have a replacement for though. It's super rare, but I'd still rather have it around.